plugins/resources/replication/irods_repl_rebalance.cpp

```diff
@@ -96,12 +96,13 @@ namespace {
         addInxVal(&genquery_inp_wrapped.get().sqlCondInp, COL_D_DATA_ID, (boost::format("= '%lld'") % _data_id).str().c_str());
 
         std::stringstream cond_ss;
+        cond_ss << "IN (";
         for (auto& bun : _leaf_bundles) {
             for (auto id : bun) {
-                cond_ss << "= '" << id << "' || ";
+                cond_ss << "'" << id << "',";
```

Comment:
using fmtlib here could improve readability.

consider:
```c++
cond_ss << fmt::format("'{}',");
```

Reply:
`bun` in the outer loop can be marked as `const`

Reply:
fmt and const, done